test/misc/Address-Resolver.specs.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from "chai";
+import { Signer } from "ethers";
+import { ethers } from "hardhat";
+import {
+  AddressResolver,
+  AddressResolver__factory,
+  SmartAccountFactory,
+  SmartAccountFactoryV1,
+  SmartAccountFactoryV1__factory,
+} from "../../typechain-types";
+
+// WIP
```

Comment:
do you want to implement the tests before merging the PR?

Reply:
PR can wait. will add test and just want to get started with development on sdk using bytecode / test deployments.